src/main/resources/exclusions.properties

```diff
@@ -150,3 +150,8 @@ spring-data-rest=skip-all
 hibernate-orm-rest-data-panache=skip-all
 ## Spring Web can only work if 'quarkus-resteasy-jackson' or 'quarkus-resteasy-reactive-jackson' is present
 funqy-knative-events,spring-web=skip-all
+## kind extension needs 'kind' tool available
+kind=skip-all
+## https://github.com/quarkusio/quarkus/issues/35476
```

Comment:
Maybe we should add TODO to not forget to remove it later?

Reply:
todos are not used here, basically all the exclusions are in TODO category, no need for that keyword